src/lcm/simulation/simulate.py

```diff
@@ -208,73 +214,64 @@ def simulate(
     return as_panel(processed, n_periods=n_periods)
 
 
-def solve_continuous_problem(
-    data_scs: StateChoiceSpace,
-    compute_ccv: Callable[..., tuple[Array, Array]],
-    vf_arr: Array,
-    params: ParamsDict,
-) -> tuple[Array, Array]:
-    """Solve the agents' continuous choice problem.
+@partial(vmap_1d, variables=("conditional_continuous_action_argmax", "discrete_argmax"))
+def _pick_optimal_continuous_actions(
```

Comment:
Unless there is a difference between "pick" and "look up", please harmonize the names.